packages/tocco-ui/src/EditableValue/EditableValue.stories.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import {storiesOf} from '@storybook/react'
+import {withKnobs, number, text, boolean} from '@storybook/addon-knobs'
+
+import EditableValue from './'
+import {storybookHtmlMarkup} from '../util/storybookHtmlMarkup'
+import {SelectStory} from '../Select/Select.stories'
+import {UploadStory} from '../Upload/Upload.stories'
+import {Upload} from '../Upload/Upload'
+import {EditableValueStory} from './EditableValueStory.stories'
+import StringEdit from './typeEditors/StringEdit'
+import TextEdit from './typeEditors/TextEdit'
+import UrlEdit from './typeEditors/UrlEdit'
+import PhoneEdit from './typeEditors/PhoneEdit'
+import NumberEdit from './typeEditors/NumberEdit'
+import IntegerEdit from './typeEditors/IntegerEdit'
+import MoneyEdit from './typeEditors/MoneyEdit'
+import TimeEdit from './typeEditors/TimeEdit'
+import SingleSelect from './typeEditors/SingleSelect'
+import MultiSelect from './typeEditors/MultiSelect'
+import RemoteSelect from './typeEditors/RemoteSelect'
+import MultiRemoteSelect from './typeEditors/MultiRemoteSelect'
+import {DateEdit} from './typeEditors/DateEdit'
+import {DateRangeEdit} from './typeEditors/DateRangeEdit'
+import {DateTimeEdit} from './typeEditors/DateTimeEdit'
+import DurationEdit from './typeEditors/DurationEdit'
+import HtmlEdit from './typeEditors/HtmlEdit'
+
+const EMPTY_FUNC = () => {}
+
+storiesOf('Edit Data/EditableValue', module)
+  .addDecorator(withKnobs)
+  .add(
+    'String',
+    () =>
+      <EditableValueStory ChildComponent={StringEdit} value={'Simple String'} />,
+    {info: {propTables: [StringEdit], propTablesExclude: [EditableValueStory], source: false}}
+  )
+  .add(
+    'Text',
+    () => <EditableValueStory ChildComponent={TextEdit} value={'This is the first line.\nTo be continued...'} />,
+    {info: {propTables: [TextEdit], propTablesExclude: [EditableValueStory], source: false}}
+  )
+  .add(
+    'Url',
+    () => <EditableValueStory ChildComponent={UrlEdit} value={'http://www.tocco.ch'} />,
+    {info: {propTables: [UrlEdit], propTablesExclude: [EditableValueStory], source: false}}
+  )
+  .add(
+    'Phone',
+    () => <EditableValueStory ChildComponent={PhoneEdit} value={'+41761234567'} />,
+    {info: {propTables: [PhoneEdit], propTablesExclude: [EditableValueStory], source: false}}
+  )
+  .add(
+    'Number',
+    () => <EditableValueStory
+      ChildComponent={NumberEdit}
+      value={12345} options={{prePointDigits: 8, postPointDigits: 2}}
```

Comment:
i think all `option` properties can the shown as knobs?
Would be nice to test prePointDigits and so on.